src/Security/Authentication/test/ClaimActionTests.cs

```diff
@@ -17,10 +14,7 @@ public class ClaimActionTests
         [Fact]
         public void CanMapSingleValueUserDataToClaim()
         {
-            var userData = new JObject
-            {
-                ["name"] = "test"
-            };
+            var userData = JsonDocument.Parse("{ \"name\": \"test\" }");
```

Comment:
That's sad we have to do that. Is a "mutable" `JsonDocument` API planned for future versions of the new JSON stack?